scripts/pre-week.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+export NVM_DIR="/root/.nvm"
```

Comment:
The NVM stuff is here and in the post week. Is there a way to move it into a separate file that we source or run as part of the other scripts?